src/exception_asm.S

```diff
@@ -46,7 +43,6 @@ _v_sp0_fiq:
 .globl _v_sp0_serr
 .type _v_sp0_serr, @function
 _v_sp0_serr:
-    msr pan, #0
```

Comment:
This has to be the first thing before any stack/mem accesses, it's here and duplicated for a reason. It won't work if you move it like this.

Reply:
Well then this tricky, guess I will need to do runtime code patching during init.